ContextualPopupDecorator/tests/ContextualPopupDecorator-specs.js

```diff
@@ -1,77 +1,95 @@
-import {mount} from 'enzyme';
 import {FloatingLayerDecorator} from '@enact/ui/FloatingLayer';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
 import {ContextualPopupDecorator} from '../ContextualPopupDecorator';
 import Button from '../../Button';
 
 const ContextualButton = ContextualPopupDecorator(Button);
+
 describe('ContextualPopupDecorator Specs', () => {
-	test('should bind callback to instance', () => {
-		let called = false;
-		// Create a context with a prop on it we can use to detect access to this prop inside the
-		// instance method.
-		class CallbackInterceptor {
-			get containerNode () {
-				called = true;
-				return null;
-			}
-		}
-
-		const subject = mount(
-			<ContextualButton
-				popupComponent={Button}
-			>
-				Button
-			</ContextualButton>
```

Comment:
Note to next reviewer: This is a kind of test which refers to implementation detail of a component. A user does not see this action happening.